day1/1/my_try.py

```diff
@@ -22,13 +22,11 @@ def word_list(f):
     parser.add_argument('path', help='file path to be read')
     args=parser.parse_args()
     
-    try:
-        f = open(args.path)
-    except IOError:
-        print("Can't load file",args.path)
-        exit(1)
+        
+    with open(args.path, 'r') as f:
```

Comment:
Aggiungi la `try` prima del `with`, per gestire il caso in cui non sia possibile accedere al file